src/components/Storage/api/useSelectedFile.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react';
+
+import { StorageFile, StorageItem } from '../types';
+
+export function useSelectedFile(
```

Comment:
This hook is overkill. We just need to keep `fullPath` in state at its only call site `StorageCard`.

Reply:
Sounds reasonable to me. I'd still suggest keeping only `fullPath` and not the full object in state, but that is lower priority.